pkg/kubelet/rkt/sync.go

```diff
@@ -0,0 +1,97 @@
+/*
+Copyright 2015 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rkt
+
+import (
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	kubecontainer "github.com/GoogleCloudPlatform/kubernetes/pkg/kubelet/container"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/probe"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/types"
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/volume"
+	"github.com/golang/glog"
+)
+
+// SyncPod makes
+func (r *Runtime) SyncPod(runningPod kubecontainer.Pod, pod *api.Pod, podStatus api.PodStatus, podVolumes map[string]volume.Volume) error {
+	podFullName := kubecontainer.GetPodFullName(pod)
+	uid := pod.UID
+
+	if len(runningPod.Containers) == 0 {
+		glog.V(4).Infof("Pod is not running, start it")
```

Comment:
does the running pod has an identity/description at this point? The log is very vague.